<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LTSView</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ltsview.qrc">
    <normaloff>:/ltsview/icons/ltsview.ico</normaloff>:/ltsview/icons/ltsview.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="exportBitmap"/>
     <addaction name="exportText"/>
     <addaction name="exportVector"/>
    </widget>
    <addaction name="open"/>
    <addaction name="openTrace"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="resetViewpoint"/>
    <addaction name="separator"/>
    <addaction name="zoomIntoAbove"/>
    <addaction name="zoomIntoBelow"/>
    <addaction name="zoomOut"/>
    <addaction name="separator"/>
    <addaction name="displayStates"/>
    <addaction name="displayTransitions"/>
    <addaction name="displayBackpointers"/>
    <addaction name="displayWireframe"/>
    <addaction name="separator"/>
    <addaction name="settings"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="select"/>
    <addaction name="pan"/>
    <addaction name="zoom"/>
    <addaction name="rotate"/>
    <addaction name="separator"/>
    <addaction name="information"/>
    <addaction name="simulation"/>
    <addaction name="mark"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="openTrace">
   <property name="text">
    <string>Open &amp;Trace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="exportBitmap">
   <property name="text">
    <string>&amp;Bitmap...</string>
   </property>
  </action>
  <action name="exportText">
   <property name="text">
    <string>&amp;Text...</string>
   </property>
  </action>
  <action name="exportVector">
   <property name="text">
    <string>&amp;Vector...</string>
   </property>
  </action>
  <action name="resetViewpoint">
   <property name="text">
    <string>&amp;Reset Viewpoint</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="zoomIntoAbove">
   <property name="text">
    <string>Zoom Into &amp;Above</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="zoomIntoBelow">
   <property name="text">
    <string>Zoom Into &amp;Below</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="zoomOut">
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="displayStates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display &amp;States</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="displayTransitions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display &amp;Transitions</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="displayBackpointers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Back&amp;pointers</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="displayWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display &amp;Wireframe</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="settings">
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
  </action>
  <action name="select">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Select</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="pan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Pan</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="zoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Zoom</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="rotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Rotate</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="information">
   <property name="text">
    <string>&amp;Information...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="simulation">
   <property name="text">
    <string>Sim&amp;ulation...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="mark">
   <property name="text">
    <string>&amp;Mark...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ltsview.qrc"/>
 </resources>
 <connections/>
</ui>
